python/requirements.txt

```diff
@@ -8,7 +8,7 @@ google
 matplotlib
 nose
 nilearn
-sklearn
+scikit-learn
```

Comment:
why is that changed?

Reply:
The 'sklearn' PyPI package is deprecated, use 'scikit-learn'
      rather than 'sklearn' for pip commands.